docs/source/getting_started/index.rst

```diff
@@ -4,4 +4,49 @@
 Introduction
 ============
 
-TODO: Do this section.
\ No newline at end of file
+This article will help you figure out how to start using Ink with GDScript in Godot.
+
+What is Inkgd?
+==============
+
+Inkgd is Ink for Godot, more precisely GDScript. It's a GDScript API that you import as an addon in your Godot project that can read an Ink story (file with .ink.json suffix) and execute Ink's methods directly in GDScript. With Inkgd you'll be able to load your story and get the text from each knot, choice and diverts, then use the text as-is. It also binds variable changes and external functions with custom functions found in your Godot scripts.
+
+A normal use of Inkgd would be to load a compiled story in '.ink.json' and then call "continue_story" until a branch is found. Here the choices are read as an array of strings and the story waits for you to input a choice. Once you submit the selected choice (represented by the index of the choices array) to the story, you repeat the process (i.e. keep calling "continue_story" until a branch is found).
```

Comment:
I feel an inline ` ``continue_story`` ` would look slightly better than `"continue_story"`, what do you think? (Unless that's not supported)

Reply:
Yeah using backticks here sounds alright, it's the name of the method as well so I think is ok.